index.html

```diff
@@ -123,21 +136,85 @@
     // SCALES
 
     x = d3.scale.linear()
-      .domain([0, d3.max(data, function (d) { return d.unemployment; })])
+      .domain(d3.extent(data, function (a) {return a.year; }))
       .range([0, width])
       .nice();
 
     y = d3.scale.linear()
-      .domain(d3.extent(data, function (d) { return d.inflation; }))
+      .domain(d3.extent(data, function (a) {return a.value; }))
       .range([height, 0])
       .nice();
 
```

Comment:
For both the scales above, you need:

var x = 
and 
var y =

not just

x =
and 
y =